metaflow/plugins/project_decorator.py

```diff
@@ -13,6 +13,19 @@
 
 
 class ProjectDecorator(FlowDecorator):
+    """
+    Specifies what projects belong to the same project.
+
+    A project-specific namespace is created for all flows that
+    use the same `@project(name)`.
+
+    Parameters
+    ----------
+    name : str
+        Project name. Make sure that the name is unique amongst all
```

Comment:
nit: maybe specify valid value for project names?

Reply:
good point, fixed